packages/scene-graph-mediator/runtime/src/importer/Pixi.ts

```diff
@@ -322,4 +329,15 @@ export default class Pixi extends Importer {
       }
     });
   }
+
+  protected createRuntimeObjectForPlugins(node: Node, resources: any): any | null {
```

Comment:
This could be moved to Importer isn't it ?

Reply:
Well, yes, that's right.

Reply:
@dolow 
fixed.
c9283caeabfe1d0f590c1b0aebc7fbef749d25bc